README.md

```diff
@@ -72,6 +72,18 @@ Example commands:
 - GitHub: [@githubhandle](https://github.com/mershark)
 - LinkedIn: [Profile link](https://www.linkedin.com/in/mershark/)
 
+👤 **Mozamel Jawad**
+
+- GitHub: [@githubhandle](https://github.com/MozamelJawad)
+- Twitter: [@twitterhandle](https://twitter.com/mozameljawad)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mozamel-jawad/)
+
+👤 **Young Gaius**
+
+- Github: [Young-Gaius](https://github.com/Young-Gaius)
+- Linkedin: [Young-gaius](https://www.linkedin.com/in/young-gaius-205624268/)
+- Twitter: [Young Gaius](https://twitter.com/EhuYoung)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
```

Comment:
![image](https://github.com/mershark/database-performance-audit/assets/93039359/96d4d58c-a72b-452d-ba20-c030fcf6d71e)

- [ ] It would be nice if you fix the highlighted link in the image above ⬆️ because it is broken 😨 Kindly fix it or remove it if you don't have any dedicated section for this link 🤔